### DISCUSSION TITLE
How to use the `pb_public` folder

### DISCUSSION THREAD

**Discussion by myCodebox** [pocketbase/pocketbase#132] [Q&A] [answered]
First, thank you for this cool project! 

When i read it right, it is enough to create a folder `pb_public` alongside the executable. I put a `index.htm` and a image in this folder

```
+ pocketpase_root
 |+ pb_data
 |+ pb_public
  |- index.html
  |- image.jpg
 |- LICENSE.md
 |- pocketbase.exe
```
I start the server with:
```sh
pocketpase_root\pocketbase.exe serve --http="127.0.0.1:80"
```

But when i try to open `127.0.0.1`, `127.0.0.1/image.jpg` or `127.0.0.1/index.html` i get an 404.

```json
{
  "code": 404,
  "message": "Not Found.",
  "data": {}
}
```

Do i something wrong or have i misinterpret the use case of this folder?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, that's the use case for this folder. The above should work; you don't even need  to specify the index.html - it should be loaded by default.

Maybe is a compatibility issue with Windows, I'll have to verify it.

**Reply by ganigeorgiev** [reply]
Just tested it on Windows 10 21H2 and it works as expected for me, aka. the files in `pb_public` are served when trying to access them.

I'm not sure why is failing in your case. You could check the permissions to make sure that the user executing the `serve` command can read the directory and files. Alternatively, you could also start PocketBase with `--debug` and check in the console if there is a error log when you try to access your pb_public files.

**Comment by myCodebox** [comment]
Thanks for testing!

I started it from WSL1 (Debian 11) and everything works as it should. After that, i used the same folder structure with the `.exe` and it now runs on WIN11 too.

Unfortunately, I can't reproduce it.

I'll keep an eye on it and if I notice anything, I'll be happy to let you know.

**Comment by kigiri** [comment]
If you use as a framework, you have to specify that you want to serve the files like in https://github.com/pocketbase/pocketbase/blob/b8c54568e36b3c59bde30a8cc2be422efc524973/examples/base/main.go#L36-L41

This was what I was missing personally :)
